### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by amit-s19**

### CODE REVIEW

File: src/pages/Home/index.jsx [hunk 1]
```diff
@@ -47,20 +54,24 @@ const Home = () => {
   const handleSubmit = async (event) => {
     if (apiCall) return;
     event.preventDefault()
-    setApiCall(true);
-
     setUsernameError(false)
     setPasswordError(false)
 
-    if (username == '') {
```
Comment (geeky-abhishek) [inline src/pages/Home/index.jsx:55]: I think we should not allow capital letters here

File: src/pages/Home/index.jsx [hunk 2]
```diff
@@ -47,20 +54,24 @@ const Home = () => {
...
+    if (username == '' || !/^[a-zA-Z]{2}_\d+$/.test(username)) {
       setUsernameError(true)
+      setTimeout(() => {
```
Comment (geeky-abhishek) [inline src/pages/Home/index.jsx:62]: remove it using onChange function

File: src/pages/Home/index.jsx (hunk 1 above)
Comment (amit-s19) [reply src/pages/Home/index.jsx:55]: Done

File: src/pages/Home/index.jsx (hunk 2 above)
Comment (amit-s19) [reply src/pages/Home/index.jsx]: Done

**Pull request merged**